apps/fares/test/month_test.exs

```diff
@@ -0,0 +1,335 @@
+            name: {:zone, "4"},
+            price_label: nil,
+            reduced: nil
+          }
+        ]
+      end
+
+      assert %Fare{cents: 27_100} =
+               Month.lowest_pass(route, nil, origin_id, destination_id, fare_fn)
+
+      assert %Fare{cents: 28_100} =
+               Month.highest_pass(route, nil, origin_id, destination_id, fare_fn)
+    end
+
+    test "returns an interzone fares that are not discounted for a trip that does not originate/terminate in Zone 1A" do
+      route = %Route{type: 2}
+      origin_id = "Ballardvale"
+      destination_id = "Haverhill"
+
+      fare_fn = fn @default_filters ++ [name: {:interzone, "4"}] ->
+        [
+          %Fare{
+            additional_valid_modes: [:bus],
+            cents: 13_900,
+            duration: :month,
+            media: [:commuter_ticket],
+            mode: :commuter_rail,
+            name: {:interzone, "4"},
+            price_label: nil,
+            reduced: nil
+          },
+          %Fare{
+            additional_valid_modes: [],
+            cents: 12_900,
+            duration: :month,
+            media: [:mticket],
+            mode: :commuter_rail,
+            name: {:interzone, "4"},
+            price_label: nil,
+            reduced: nil
+          }
+        ]
+      end
+
+      assert %Fare{cents: 12_900} =
+               Month.lowest_pass(route, nil, origin_id, destination_id, fare_fn)
+
+      assert %Fare{cents: 13_900} =
+               Month.highest_pass(route, nil, origin_id, destination_id, fare_fn)
+    end
+
+    test "returns zone-based fares for standard trips on Foxboro pilot" do
+      route = %Route{type: 2, id: "CR-Franklin"}
+      trip_1 = %Trip{name: "751", id: "CR-Weekday-Fall-19-751"}
+      trip_2 = %Trip{name: "759", id: "CR-Weekday-Fall-19-759"}
+
+      assert %Fare{name: {:zone, "4"}} =
+               Month.lowest_pass(route, trip_1, "place-sstat", "place-FS-0049")
+
+      assert %Fare{name: {:interzone, "3"}} =
+               Month.lowest_pass(route, trip_2, "place-FB-0118", "place-FS-0049")
+    end
+
+    test "accepts a Trip ID" do
+      route = %Route{type: 2, id: "CR-Franklin"}
+      trip_id = "CR-Weekday-Fall-19-751"
+
+      assert %Fare{name: {:zone, "4"}} =
+               Month.lowest_pass(route, trip_id, "place-sstat", "place-FS-0049")
+
+      assert %Fare{name: {:zone, "4"}} =
+               Month.highest_pass(route, trip_id, "place-sstat", "place-FS-0049")
+    end
+
+    test "returns nil if no matching fares found" do
+      route = %Route{type: 2, id: "CapeFlyer"}
+      origin_id = "place-sstat"
+      destination_id = "Hyannis"
+
+      fare_fn = fn _ -> [] end
+
+      assert Month.lowest_pass(route, nil, origin_id, destination_id, fare_fn) == nil
+    end
+  end
+
+  describe "ferry" do
+    test "returns the fares that are not discounted for the correct ferry trip" do
+      route = %Route{type: 4}
+      origin_id = "Boat-Charlestown"
+      destination_id = "Boat-Long-South"
```

Comment:
Not sure it matters, but our current ferry IDs are Boat-F1 and Boat-F4

Reply:
Are those the IDs for the routes or for the stops (or both)?

Reply:
Ahh right nevermind, you're talking stops and I'm talking routes. Sorry!